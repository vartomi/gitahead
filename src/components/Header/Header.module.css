.container {
    background-color: #24292e;
    padding: 16px;
    height: 64px;
    overflow: auto;
    color: white;
    display: flex;
    align-items: center;
}

.container > h3 {
    margin: 0 16px;
}

.logo {
    fill: white;
}