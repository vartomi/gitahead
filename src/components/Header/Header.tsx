import React from 'react';
import styles from './Header.module.css';
import { ReactComponent as Logo } from '../../assets/logo.svg';

const Header: React.FC = () => {
  return (
    <header className={styles.container}>
      <Logo className={styles.logo} /><h3>GitAhead</h3>
    </header >
  );
}

export default Header;
