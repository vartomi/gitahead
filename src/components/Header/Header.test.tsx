import { shallow } from 'enzyme'
import Header from './Header';
import React from 'react';

describe('Header component', () => {
    it('should be rendered with h3 title', () => {
        const wrapper = shallow(<Header />)
        expect(wrapper.find('h3').text()).toEqual('GitAhead');
    });

    it('should render snapshot', () => {
        const wrapper = shallow(<Header />)
        expect(wrapper).toMatchSnapshot();
    });
});