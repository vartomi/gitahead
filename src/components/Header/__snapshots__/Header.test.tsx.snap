// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render snapshot 1`] = `
<header
  className="container"
>
  <ForwardRef(SvgLogo)
    className="logo"
  />
  <h3>
    GitAhead
  </h3>
</header>
`;
